src/battle_main.c

```diff
@@ -3842,10 +3842,12 @@ static void HandleTurnActionSelectionState(void)
                     }
                     break;
                 case B_ACTION_USE_ITEM:
-                    if (gBattleTypeFlags & (BATTLE_TYPE_LINK
+                    if ((gBattleTypeFlags & (BATTLE_TYPE_LINK
                                             | BATTLE_TYPE_FRONTIER_NO_PYRAMID
                                             | BATTLE_TYPE_EREADER_TRAINER
                                             | BATTLE_TYPE_RECORDED_LINK))
+						// Or if currently held by Sky Drop
+						|| ((gStatuses3[gActiveBattler] & STATUS3_ON_AIR) && (gStatuses3[gActiveBattler] & STATUS3_UNDERGROUND)))
```

Comment:
Use empty spaces instead of tabs so the conditions you added to the check are properly aligned with the ones that were already there.